k8s-secrets.tf

```diff
@@ -0,0 +1,15 @@
+resource "kubernetes_secret" "hms_secrets" {
+  count = "${var.hms_instance_type == "k8s" ? 1 : 0}"
```

Comment:
This piece gets a bit nebulous to me, i.e. if we aren't using an external database what should these secrets be?

Reply:
external is used when apiary terraform module in not managing database, so we shouldn't need this when using external database.